pkgs/development/node-packages/default.nix

```
@@ -13,6 +13,21 @@ let
         export NG_CLI_ANALYTICS=false
       '';
     };
+
+    aws-azure-login = super.aws-azure-login.override (
+      if stdenv.isDarwin
+      then {}
+      else {
```

```suggestion
     {
```

this is not valid fox nixpkgs. Please use broken or platform instead.

---

Thanks. I will open a separate PR to fix the other instances in this file then too.